css/style.css

```
@@ -0,0 +1,411 @@
+.wrapper {
+  position: relative;
+}
+.wrapper .slider-main {
+  position: absolute;
+  top: 0rem;
+  bottom: 0rem;
+  left: 0rem;
+  right: 0rem;
+  width: 100%;
+  height: 95.9rem;
+  z-index: 2;
+  background-color: #2c2e3d;
+  max-width: 192rem;
+  margin: 0 auto;
+  position: relative;
```

Đoạn này css như nào ấy nhỉ. có cả position: absolute; và position: relative là sao nhỉ
Thêm nữa left = 0. right = 0 thì cần gì width: 100% và max-width, và fix cứng height ở đây làm gì nhỉ

---

1. Element này dùng Relative là đúng, em quên chưa bỏ Absolute và top-left-bottom-right đi
2. Vì dùng Relative nên cần có Height
3. Max-width thì đúng là bỏ ạ

```suggestion

.wrapper .slider-main {
  position: relative;
  width: 100%;
  height: 95.9rem;
  margin: 0 auto 20rem;
  background-color: #2c2e3d;
  overflow: hidden;
}

```

---

2: dùng absolute chứ bên ngoài có relative r mà

---

Cái Wrapper ở ngoài ko cần Position, ban đầu em định cho Slider__arrow ăn theo thằng Wrapper đó, nhưng để ăn theo thằng Slider-main là được r